docs/integrations/dbt.md

```diff
@@ -8,9 +8,11 @@ SQLMesh has native support for running dbt projects with its dbt adapter.
 Prepare an existing dbt project to be run by SQLMesh by executing the `sqlmesh init` command *within the dbt project root directory* and with the `dbt` template option:
 
 ```bash
-$ sqlmesh init -t dbt
+$ sqlmesh init <default SQL dialect> -t dbt
```

Comment:
The default dialect is determined from the profile. dbt users should specify.

Reply:
Should not specify?

Reply:
shouldn't, correct.